src/Jellyfin.Database/Jellyfin.Database.Providers.Sqlite/SqliteDatabaseProvider.cs

```diff
@@ -58,7 +58,8 @@ public void Initialise(DbContextOptionsBuilder options, DatabaseConfigurationOpt
                 sqLiteOptions => sqLiteOptions.MigrationsAssembly(GetType().Assembly))
             // TODO: Remove when https://github.com/dotnet/efcore/pull/35873 is merged & released
             .ConfigureWarnings(warnings =>
-                warnings.Ignore(RelationalEventId.NonTransactionalMigrationOperationWarning));
+                warnings.Ignore(RelationalEventId.NonTransactionalMigrationOperationWarning))
+            .AddInterceptors(new PragmaConnectionInterceptor(134_217_728, "NORMAL", 134_217_728, null));
```

Comment:
Wouldn't it be better to expose this in our FB settings to make it configurable? Might even make more sense to set this a default on the interceptor directly and make them overridable.